static/js/pages/instances/InstanceFormGuided.tsx

```diff
@@ -43,17 +50,29 @@ const InstanceFormGuided: FC = () => {
     instanceType: Yup.string().required("Instance type is required"),
   });
 
+  const create = () => {
+    setWithStart(false);
+    formik.handleSubmit();
+  };
+
+  const createAndStart = () => {
+    setWithStart(true);
+    formik.handleSubmit();
```

Comment:
This could fail, when the state update doesn't land in time before it is queried. It it usually better to pass the boolean directly to the function handling it. But it might be hard in this case, because handlesubmit doesn't allow argument passing easily.
I think we should split the actual logic to create and start into a function that receives the values and the boolean. Then we can call this function from here, from line 54 above and from formiks onSubmit as well. Does it make sense?